.github/workflows/release.yaml

```diff
@@ -182,19 +184,12 @@ jobs:
           cd _dist
           tar czf py2wasm-${{ env.RELEASE_VERSION }}-${{ env.RUNNER_OS }}-${{ matrix.config.arch }}.tar.gz py2wasm.license py2wasm${{ matrix.config.extension }}
 
-      - name: Upload build artifact
```

Comment:
As far as I can tell this step was unnecessary.

Reply:
This is required so that the artifact can be downloaded for the checksum step.

Reply:
Ah of course! Thanks 👍